laravel/resources/views/layout.blade.php

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="de">
```

Comment:
[CONCERN]

The entire application is currently only available in English, so this should be by default English, not German.

Additional, I would also rather use the Laravel default implementation, which is more dynamic and allows later multi-language support:
```suggestion
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
```